src/hydra/parseHydraDocumentation.test.js

```diff
@@ -1276,33 +1288,55 @@ test('Parse entrypoint class without "supportedProperty" key', () => {
 
   fetch.mockResponses([entrypoint, init], [docs, init]);
 
-  parseHydraDocumentation("http://localhost/").catch(data => {
-    expect(data.message).toBe(
-      'The entrypoint definition has no "http://www.w3.org/ns/hydra/core#supportedProperty" key or it is not an array.'
-    );
-  });
+  const expectedError = { message: "" };
+
+  try {
+    await parseHydraDocumentation("http://localhost/");
+  } catch (error) {
+    expectedError.message = error.message;
+  }
+
+  expect(expectedError.message).toBe(
+    'The entrypoint definition has no "http://www.w3.org/ns/hydra/core#supportedProperty" key or it is not an array.'
+  );
 });
 
-test("Invalid docs JSON", () => {
+test("Invalid docs JSON", async () => {
   const docs = `{foo,}`;
 
   fetch.mockResponses([entrypoint, init], [docs, init]);
 
-  parseHydraDocumentation("http://localhost/").catch(data => {
-    expect(data).toHaveProperty("api");
-    expect(data).toHaveProperty("response");
-    expect(data).toHaveProperty("status");
-  });
+  const expectedError = {};
+
+  try {
+    await parseHydraDocumentation("http://localhost/");
+  } catch (error) {
+    expectedError.api = error.api;
+    expectedError.response = error.response;
+    expectedError.status = error.status;
+  }
+
+  expect(expectedError).toHaveProperty("api");
+  expect(expectedError).toHaveProperty("response");
+  expect(expectedError).toHaveProperty("status");
 });
 
-test("Invalid entrypoint JSON", () => {
-  const entrypoint = `{foo,}`;
+test("Invalid entrypoint JSON", async () => {
+  let entrypoint = `{foo,}`;
```

Comment:
Why did you use backquote here?

Reply:
I did not change the format, but i think this is suppose to be a JSON string, not a JS Object ? (Like the [```const entrypoint```](https://github.com/jf-tilleuls/api-doc-parser/blob/8bc407cd074767f4dfb5a4032ee5e99561cf49f4/src/hydra/parseHydraDocumentation.test.js#L3))

Reply:
OK, I did not see than you "only" change `const` to `let`. Why did you change it (I did not see another assignation of `entrypoint` in the test)?